docs/maps/map-settings.asciidoc

```diff
@@ -0,0 +1,38 @@
+[role="xpack"]
+[[maps-settings]]
+== Map settings
+
+To configure map settings, click *Map settings* in the application toolbar.
```

Comment:
Although the intro text is ok as is, I think it might be better to add a little more information. 

Some suggestions:

Elastic Maps offers settings that let you configure how a map is displayed.  To access these settings, click *Map settings" in the application toolbar.

Looking to customize your map? Simply click *Map settings" in the application toolbar. You'll be able to configure zoom and spatial filters.